hw/xbox/nv2a/swizzle.c

```diff
@@ -102,27 +116,26 @@ void swizzle_box(
     for (z = 0; z < depth; z++) {
         for (y = 0; y < height; y++) {
             for (x = 0; x < width; x++) {
-                const uint8_t *src = src_buf
-                                         + y * row_pitch + x * bytes_per_pixel;
-                uint8_t *dst = dst_buf + get_swizzled_offset(x, y, 0,
-                                                             mask_x, mask_y, 0,
-                                                             bytes_per_pixel);
+                const uint8_t *src = src_buf + y * row_pitch +
+                                     x * bytes_per_pixel;
+                uint8_t *dst = dst_buf +
+                               get_swizzled_offset(
+                                   x, y, 0, mask_x, mask_y, 0, bytes_per_pixel);
```

Comment:
Also harder to read now in my opinion.

Reply:
Got wrapped for 80 column limit, I think it looks okay though?